src/test/ui/editions/edition-keywords-2018-2018-parsing.stderr

```diff
@@ -14,19 +14,24 @@ error: no rules expected the token `r#async`
   --> $DIR/edition-keywords-2018-2018-parsing.rs:22:31
    |
 LL |     r#async = consumes_async!(r#async); //~ ERROR no rules expected the token `r#async`
-   |                               ^^^^^^^ no rules expected the token `r#async`
+   |                               ^^^^^^^ no rules expected this token in macro call
 
 error: no rules expected the token `async`
   --> $DIR/edition-keywords-2018-2018-parsing.rs:23:35
    |
 LL |     r#async = consumes_async_raw!(async); //~ ERROR no rules expected the token `async`
-   |                                   ^^^^^ no rules expected the token `async`
+   |                                   ^^^^^ no rules expected this token in macro call
 
-error: expected one of `move`, `|`, or `||`, found `<eof>`
-  --> <::edition_kw_macro_2018::passes_ident macros>:1:22
+error: expected one of `move`, `|`, or `||`, found the end of the macro arm
```

Comment:
*Man,* it still took me a long time to understand this message, though I agree it is definite progress. At this point it's just wording though.

Reply:
What about:

```
macro expansion ends with an incomplete expression: expected one of `move`, `|`, or `||`
```

similarly we could print

```
file ends with an incomplete expression: expected one of `move`, `|`, or `||`
```

I guess though we don't know it's an expression. I'm trying to think if there is some more generic term that would be suitable.

Reply:
I believe the first one is a bit easier to understand. Pushed change using that wording. Let me know what you think.